tf_agents/cogment_verse_tf_agents/reinforce/reinforce_agent_adapter.py

```diff
@@ -43,38 +41,13 @@ class ReinforceAgentAdapter(AgentAdapter):
     def _create(self, model_id, **kwargs):
         return ReinforceAgent(id=model_id, **kwargs)
 
-    @staticmethod
-    def extract_agent_params(model):
-        agent_params = {}
-        agent_params["id"] = model.id
-        agent_params["_params"] = model._params
-        agent_params["_epsilon_schedule"] = model._epsilon_schedule
-        agent_params["_lr_schedule"] = model._lr_schedule
-        agent_params["model_params"] = model._model.get_weights()
-        return agent_params
-
     def _load(self, model_id, version_number, version_user_data, model_data_f):
-
-        agent_params = torch.load(model_data_f)
-        reinforce_agent = ReinforceAgent(
-            id=agent_params["id"],
-            obs_dim=agent_params["_params"]["obs_dim"],
-            act_dim=agent_params["_params"]["act_dim"],
-            epsilon_schedule=agent_params["_epsilon_schedule"],
-            lr_schedule=agent_params["_lr_schedule"],
-            gamma=agent_params["_params"]["gamma"],
-            max_replay_buffer_size=agent_params["_params"]["max_replay_buffer_size"],
-            seed=agent_params["_params"]["seed"],
-        )
-
-        reinforce_agent.init_params(agent_params["model_params"])
-
+        reinforce_agent = ReinforceAgent(id=model_id)
```

Comment:
You are doing two initialization of nets here, which is a (minor) waste. Just make load static

Reply:
I am not sure how making load static would resolve the (minor) re-initialization step.